Vagrantfile

```diff
@@ -15,7 +15,8 @@ def packages_debianoid(user)
     apt-get -y -qq update
     apt-get -y -qq dist-upgrade
     # for building borgbackup and dependencies:
-    apt install -y libssl-dev libacl1-dev libxxhash-dev liblz4-dev libzstd-dev pkg-config
+    apt install -y pkg-config
+    apt install -y libssl-dev libacl1-dev libxxhash-dev liblz4-dev libzstd-dev || true
```

Comment:
guess this will fail to install the remainder of the pkgs, if libxxhash-dev can not be found (like on stretch).

Reply:
Yes, it fails then for all of them. The locally built libs take priority in pkg-config and the DPKG variants would not be used, when `install source dependencies` is active.

Reply:
ok